icu4c/source/test/intltest/intltest.cpp

```diff
@@ -20,6 +20,7 @@
 #include <string.h>
 #include <cmath>
 #include <math.h>
+#include <sys/stat.h>
```

Comment:
might be more portable to just try to read a text file as it was rather than bring in `sys/stat.h` which may not be available everywhere.

Reply:
Fixed in [b8dfad4](https://github.com/unicode-org/icu/pull/3050/commits/b8dfad4117495b896ab2f40a4df1f696173aa646)